src/components/LineSeries.tsx

```diff
@@ -65,10 +65,11 @@ export default function LineSeries(props: LineSeriesProps) {
     <g>
       {props.hidden ? null : <LineSeriesRender {...lineProps} />}
       <ErrorBars
+        {...props.errorBars}
         xAxis={props.xAxis}
         yAxis={props.yAxis}
         data={props.data}
-        {...props.errorBars}
+        hidden={props.errorBars?.hidden || props.hidden}
```

Comment:
can you please keep the original spread and instead put the "props.hidden" condition one level higher ? i.e. render `null` instead of the group. Then you can simplify <LineSeriesRender and <ScatterSeries too.